lavalink/PlayerManager.py

```diff
@@ -218,11 +224,19 @@ def find_all(self, predicate):
     def get(self, guild_id):
         """ Returns a player from the cache, or creates one if it does not exist. """
         if guild_id not in self._players:
-            p = self._player(lavalink=self.lavalink, guild_id=guild_id)
+            p = self._player(node=self.node, lavalink=self.lavalink, guild_id=guild_id)
             self._players[guild_id] = p
 
         return self._players[guild_id]
 
+    async def clear_guild(self, guild_id):
+        """ Removes the player and clears the cache. """
+        try:
+            await self._players[guild_id].disconnect()
+            del self._players[guild_id]

```

Comment:
Very similar to `remove()`
It can be a good idea to mix both or to call `remove()` in `clear_guild()`

Reply:
indeed, it's more readable

Reply:
or we could just pop it out?

Reply:
Wut ? 

I was talking about this [remove](https://github.com/Devoxin/Lavalink.py/blob/3d96f5729ff6dc85466fd652824455f2f23410d7/lavalink/PlayerManager.py#L240), sorry if I wasn't clear

Reply:
i think that's more future proof, but what is cleanup method hmm
```python
async def clear_guild(self, guild_id):
    """ Removes the player and clears the cache. """
    try:
        player = self._players.pop(guild_id)
        await player.disconnect()
    except KeyError:
        pass
```

Reply:
I think clear_guild should be removed in favour of remove, because disconnects should be handled anyway.